pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -318,10 +318,10 @@ func (p *packetParser) clean(tcnl ITc, tcIngressObj *tc.Object, tcEgressObj *tc.
 	// Warning, not error. Clean is best effort.
 	if tcnl != nil {
 		if err := getQdisc(tcnl).Delete(tcEgressObj); err != nil && !errors.Is(err, tc.ErrNoArg) {
```

Comment:
We should check if the error is because the`veth` was deleted. If that's the case, it's not an error.